source/beamline/BioXAS/BioXASBeamline.h

```diff
@@ -418,14 +408,9 @@ protected slots:
 	void addControlAsDetector(const QString &name, const QString &description, AMControl *control, bool hiddenFromUsers = false, bool isVisible = true);
 
 	/// Protected constructor.
-	BioXASBeamline(const QString &controlName);
+	BioXASBeamline(const QString &beamlineName, const QString &controlName);
```

Comment:
I think he means when being used in warning message boxes or in the status view.